command/operator_autopilot_set.go

```diff
@@ -73,18 +46,22 @@ func (c *OperatorAutopilotSetCommand) Run(args []string) int {
 		return 1
 	}
 
-	// Update the config values based on the set flags.
-	cleanupDeadServers.Merge(&conf.CleanupDeadServers)
-	enableRedundancyZones.Merge(&conf.EnableRedundancyZones)
-	disableUpgradeMigration.Merge(&conf.DisableUpgradeMigration)
-	enableCustomUpgrades.Merge(&conf.EnableCustomUpgrades)
-
-	trailing := uint(conf.MaxTrailingLogs)
-	maxTrailingLogs.Merge(&trailing)
-	conf.MaxTrailingLogs = uint64(trailing)
-	minQuorum.Merge(&conf.MinQuorum)
-	lastContactThreshold.Merge(&conf.LastContactThreshold)
-	serverStabilizationTime.Merge(&conf.ServerStabilizationTime)
+	f := c.Meta.FlagSet("autopilot", FlagSetClient)
+	f.Usage = func() { c.Ui.Output(c.Help()) }
+
+	f.BoolVar(&conf.CleanupDeadServers, "cleanup-dead-servers", conf.CleanupDeadServers, "")
+	f.Uint64Var(&conf.MaxTrailingLogs, "max-trailing-logs", conf.MaxTrailingLogs, "")
+	f.DurationVar(&conf.LastContactThreshold, "last-contact-threshold", conf.LastContactThreshold, "")
+	f.DurationVar(&conf.ServerStabilizationTime, "server-stabilization-time", conf.ServerStabilizationTime, "")
+	f.BoolVar(&conf.EnableRedundancyZones, "enable-redundancy-zones", conf.EnableRedundancyZones, "")
+	f.BoolVar(&conf.DisableUpgradeMigration, "disable-upgrade-migration", conf.DisableUpgradeMigration, "")
+	f.BoolVar(&conf.EnableCustomUpgrades, "enable-custom-upgrades", conf.EnableCustomUpgrades, "")
+	f.UintVar(&conf.MinQuorum, "min-quorum", conf.MinQuorum, "")
```

Comment:
This looks good to me now and I'll double check the logic - not sure why we haven't done that before, so will double check.  Thanks!

Reply:
I've reverted this back as it's not that easy, apparently.